src/SQLite3-Core-Tests/SQLite3BaseConnectionTest.class.st

```diff
@@ -344,6 +344,19 @@ SQLite3BaseConnectionTest >> testBooleanWriteThenRead [
 		]
 ]
 
+{ #category : #'tests - connections' }
+SQLite3BaseConnectionTest >> testCloseFile [
+
+	| fileName newConnection path |
+	path := Smalltalk os isWindows ifTrue: [ (Smalltalk os environment at: 'TMP'), '\' ] ifFalse: [ '/tmp/' ].
```

Comment:
- extract into an own method #getTempFolder or similar

Reply:
Done in the test class for now. I'll look at moving this to OSPlatform for Pharo 12.